### ISSUE TITLE
Macbook Pro 17 inch mid 2010

### ISSUE THREAD

**Issue by skiabox** [Floris497/mac-pixel-clock-patch-V2#175]
Hello sir.
I have this macbook pro and my monitor is LG 29UM58.
I managed to have my monitor run at 2560x1080@52Hz using SwitchResX.
Do you believe that I can run this resolution (2560x1080) at 60Hz?
Does it make any difference?
I am using the monitor for coding.
Thank you very much!

**Comment by Floris497** [comment]
Are you using HDMI? that should be around the limit if what your HDMI port can output.. would recommend using DP.. but not really sure..

**Comment by Floris497** [comment]
And actually 50Hz isn't that bad you can use it like this, it's way better than 30

**Comment by skiabox** [comment]
Thank you for your rapid answer!
Yes I am using hdmi and I was thinking if I can get 60Hz using a newer adaptor.
I see a lot of newer adaptors (for example active adaptors) in Amazon.

**Comment by Floris497** [comment]
DP to HDMI might work but can't say for sure..

**Comment by skiabox** [comment]
Dp to hdmi (I believe that you mean mini displayport with dp) is what I am using right now but maybe I have an adaptor limitation.
Check this page.
http://monitorinsider.com/cable_bandwidth.html

**Comment by Floris497** [comment]
You need at least HDMI1.3 on the adapter, the mDP should be able to handle

**Comment by skiabox** [comment]
I have this product (minidisplay port - to - hdmi)

https://www.sendstation.com/us/products/mdpadapter/

**Comment by Floris497** [comment]
It does say 1080p as max.. and what you are achieving now is close to the 165.. including some overhead.. so it might not he HDMI1.3 or newer

**Comment by skiabox** [comment]
I see.
So I believe that the first step I must take is to buy a newer active dp-to-hdmi adaptor.
But which one should I choose?
I see so mane brands in amazon.

**Comment by skiabox** [comment]
So you suggest this product I presume :

http://www.club-3d.com/index.php/products/reader.en/product/displayport-12-to-hdmi-20-uhd-active-adapter.html

**Comment by Floris497** [comment]
Yes, but you don't really need 2.0 for this application. And they are quite expensive if you ask me.. but that should work don't you need mDP though?

**Comment by skiabox** [comment]
Omg you are correct!
So I must choose between these two I guess!

http://www.club-3d.com/index.php/products/reader.en/product/mini-displayport-12-to-hdmi-20-uhd-active-adapter.html

http://www.club-3d.com/index.php/products/reader.en/product/mini-displayport-12-cable-to-hdmi-20-active-adapter.html

**Comment by Floris497** [comment]
You could but once again.. but your choice to get HDMI2.0 they are more expensive.. actually maybe delock has some adapters too now

**Comment by skiabox** [comment]
Oh, I have seen delock a lot here in Greece.

Check this page : 

http://www.visionstudio.gr/prodinfo.asp?id=44033&cat=16

**Comment by Floris497** [comment]
Ναι αυτο φαινεται μια χαρα και ειναι και φθηνοτερο απο Club3D

**Comment by skiabox** [comment]
Οπ Έλληνας;
Χεχεχε
Λέω να το χτυπήσω λοιπόν μιας και αναφέρεται και σε αυτό το thread :

https://superuser.com/questions/616369/can-a-mini-displayport-to-hdmi-cable-support-resolutions-above-1920x1200

**Comment by skiabox** [comment]
Το μόνο που με προβληματίζει πλέον είναι το faq του SwitchResX.
Δες ένα απόσπασμα :

For example, the monitor will send 1920x1080 at 60Hz as well as 2560x1080 at 60Hz as its wanted resolutions.
If 2560x1080 at 60Hz doesn’t fit the Mac’s capabilities, then the Mac will only activate 1920x1080 at 60Hz.

In this case, there's no other predefined resolution claimed by the monitor that fits inside the Mac's limits.
SwitchResX doesn't push these limits. SwitchResX helps creating resolutions within the Mac's limits. 
One of the limitations of most Macs before 2014 when using a DisplayPort output is a resolution bandwidth of 165 MHz.

**Comment by skiabox** [comment]
Απλά διαβάζοντας αυτό που γράφουν και βλέποντας εδώ (http://monitorinsider.com/cable_bandwidth.html) πιστεύω ότι έχουν κάνει λάθος και ρίχνουν το φταίξιμο του HDMI v1.0 - v1.2 max clock (165Mhz) στο mini displayport.

**Comment by Floris497** [comment]
No, don't worry about switchresx the patch will take care if the 2560x1080 not being supported.
I'm not greek i have my personal translator :p

**Comment by skiabox** [comment]
This adaptor http://www.delock.de/produkte/F_658_Displayport_62611/merkmale.html solved my problem!!!

**Comment by Floris497** [comment]
:) enjoy
